httplib.h

```diff
@@ -8405,4 +8405,8 @@ inline SSL_CTX *Client::ssl_context() const {
 
 } // namespace httplib
 
+#ifdef CPPHTTPLIB_USE_POLL
```

Comment:
Could you change it to `#if defined(_WIN32) && defined(CPPHTTPLIB_USE_POLL)`?

Reply:
Done.